packages/util-dynamodb/src/marshallTypes.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { convertToAttr } from "./convertToAttr";
+import { marshall } from "./marshall";
+
+describe("marshall type discernment", () => {
+  describe("behaves as convertToAttr for non-collection values or Sets", () => {
+    it("marshals string", () => {
+      const value = "hello";
+      expect(marshall(value)).toEqual(convertToAttr(value));
+    });
+    it("marshals number", () => {
```

Comment:
nit: newline between tests for clarity
```suggestion

    it("marshals number", () => {
```